Cargo.toml

```diff
@@ -11,16 +11,17 @@ version = "0.16.3"
 rust-version = "1.72"
 
 [features]
-default = ["tracing_opentelemetry"]
+default = []
```

Comment:
I removed the default feature. I have the feeling that the user must be aware of the otel version he's using rather than have it by magic. 

Wdyt?

If you do agree i would remove `tracing_opentelemetry` feature or rename it to `latest_tracing_opentelemetry`. Revert this change otherwise